CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Allow multiple input files for the CLI  (see [issue 34](https://github.com/RMLio/yarrrml-parser/issues/34))
 - reference formulation for CSS  (see [issue 37](https://github.com/RMLio/yarrrml-parser/issues/37))
 - Allow multiple input files for the CLI  (see [issue 34](https://github.com/RMLio/yarrrml-parser/issues/34))
+- Overwrite references  (see [issue 35](https://github.com/RMLio/yarrrml-parser/issues/35))
```

Comment:
I would add another line under `Changed` that `_id` references now need to be prefixed as `\_id`, so we remember to do the major version update :)